content/en/docs/languages/java/configuration.md

```diff
@@ -1,185 +1,297 @@
+code. There are two distinct autoconfigure workflows:
+
+- [Environment variables and system properties](#environment-variables-and-system-properties)
+  interprets environment variables and system properties to create SDK
+  components, including various customization points for overlaying programmatic
+  configuration.
+- [Declarative configuration](#declarative-configuration) (**currently under
+  development**) interprets a configuration model to create SDK components,
+  which is typically encoded in a YAML configuration file.
+
+Automatically configure SDK components using with autoconfigure as follows:
+
+<!-- prettier-ignore-start -->
+<?code-excerpt "src/main/java/otel/AutoConfiguredSdk.java"?>
+```java
+package otel;
+
+import io.opentelemetry.sdk.OpenTelemetrySdk;
+import io.opentelemetry.sdk.autoconfigure.AutoConfiguredOpenTelemetrySdk;
+
+public class AutoConfiguredSdk {
+  public static OpenTelemetrySdk autoconfiguredSdk() {
+    return AutoConfiguredOpenTelemetrySdk.initialize().getOpenTelemetrySdk();
+  }
+}
+```
+<!-- prettier-ignore-end -->
+
+**NOTE:** The [Java agent](/docs/zero-code/java/agent/) automatically configures
+the SDK using this workflow, and uses it in the instrumentation it installs. All
+autoconfigure content is applicable to Java agent users.
 
-The following configuration options apply to the
-[Java agent](/docs/zero-code/java/agent/) and all other uses of the SDK.
-
-{{% alert title="System Properties and Environment Variables" color="info" %}}
-Any setting configurable with a system property can also be configured with an
-environment variable. Apply the following steps to convert a system property to
-an environment variable:
+{{% alert color="info" %}} The autoconfigure module registers Java shutdown
+hooks to shut down the SDK when appropriate. Because OpenTelemetry Java
+[uses `java.util.logging` for internal logging](/docs/languages/java/sdk/#internal-logging),
+some of that logging may be suppressed during shutdown hooks. This is a bug in
+the JDK itself, and not something under the control of OpenTelemetry Java. If
+you require logging during shutdown hooks, consider using `System.out` rather
+than a logging framework that might shut itself down in a shutdown hook, thus
+suppressing your log messages. See
+this[JDK bug](https://bugs.openjdk.java.net/browse/JDK-8161253) for more
+details. {{% /alert %}}
+
+### Environment variables and system properties
+
+Generally, autoconfigure supports properties listed in the
+[environment variable configuration specification](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/configuration/sdk-environment-variables.md#opentelemetry-environment-variable-specification),
+with occasional experimental and Java-specific additions.
+
+**NOTE:** The properties are listed below as system properties, but can also be
+set via environment variables. Apply the following steps to convert a system
+property to an environment variable:
```

Comment:
```suggestion
**NOTE:** The following properties are listed as system properties, but can also be
set using environment variables. Apply the following steps to convert a system
property to an environment variable:
```

Ditto on using the callout notation macro.

Reply:
I'm just going to remove the `**NOTE:**` and not replace with a callout. There's so much content in this that the callout starts to be an eyesore.